.github/workflows/tests.yml

```diff
@@ -16,7 +16,6 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-          - '2.7'
```

Comment:
Usually we keep Ruby versions until they cause some breaking changes. Any reason we're dropping this?

Reply:
Ruby 2.7 reached EOL support last year (Mar 2023) so we've began removing it along with this work. Also open to keeping it if the norm for these repos is to keep until they cause some breaking changes.

Reply:
It's open source so we don't really know what the downstream users use.

cc @casperisfine any opinion?

Reply:
👍🏻 Kept the Ruby 2.7 CI and changed the minimum required Ruby version to be `> 2.7`